airgun/entities/organization.py

```diff
@@ -27,6 +28,13 @@ def delete(self, entity_name):
         self.browser.handle_alert()
         view.flash.assert_no_error()
         view.flash.dismiss()
+        view = self.navigate_to(self, 'All')
```

Comment:
that part become too common over the code. We need to think about post delete procedure. Maybe create an issue for that

Reply:
Yeah, we already have #19 for that